exercises/1/src/components/NewsForm.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <div id="news-input">
+    <label>
+      <input v-model="news_input" placeholder="edit me">
+    </label>
+    <button @click="createNewsItem">create</button>
```

Comment:
Using a button like this works. But the task was to use a `<form>` and preventing it's default behavior (POSTing). This, I can not give you the point for this one. 

❌

Reply:
We traded this Sentence `Prevent the default behaviour of a <form> html element which would execute a HTTP request on submit.` like "If you use a form, take care of the redirect aspect" and not like you say "You have to use a form tag regardless what ever options are possible". So our point of view is, this sentence is more like a hint such as the sort side-effect thing.

Reply:
Then this is a weakness of the exercise description:
```
:star: Below the list is a form to add new items to the list
```
should be
```
:star: Below the list is a <form> to add new items to the list
```
indicating that a `<form>` html is required here.